commander3/src/commander.f90

```diff
@@ -255,7 +255,7 @@ program commander
      !----------------------------------------------------------------------------------
      ! Process TOD structures
 
-     if (iter > 0 .and. cpar%enable_TOD_analysis .and. (iter <= 2 .or. mod(iter,cpar%tod_freq) == 0)) then
+     if (iter > 1 .and. cpar%enable_TOD_analysis .and. (iter <= 2 .or. mod(iter,cpar%tod_freq) == 0)) then
```

Comment:
Not sure why this has been ticked up to > 1

Reply:
This is useful for when the sky model is a bit off. Added a comment to update this.